### PR TITLE
Uf 5846 add subscription api

### PR DESCRIPTION
**Pull request by foobar-1337**

### CODE REVIEW

File: src/main/resources/db/migration/V1.0__baseline_init.sql
Comment (josc04) [inline src/main/resources/db/migration/V1.0__baseline_init.sql:1]: Just curious. Are existing tables removed manually before this script is executed? No need to migrate existing content in existing tables?

Comment (foobar-1337) [reply src/main/resources/db/migration/V1.0__baseline_init.sql:1]: This is not my stuff (incoming from rebase), but I suppose no tables will be removed manually. New tables will be added by flyway and old tables will also be removed by flyway?

The tables with content (affected and disturbance) will remain intact.

**Comment by foobar-1337** [comment]
Acted on review comments

File: src/main/java/se/sundsvall/disturbance/api/SubscriptionResource.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	@ApiResponse(responseCode = "409", description = "Conflict", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+	@ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+	public ResponseEntity<Subscription> readSubscription(
+		@Parameter(name = "id", description = "Subscription ID", required = true, example = "1234") @PathVariable(name = "id") final long id) {
+
+		return ok(subscriptionService.read(id));
+	}
+
+	@PatchMapping(path = "/{id}", consumes = APPLICATION_JSON_VALUE, produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
```
Comment (mha109) [inline src/main/java/se/sundsvall/disturbance/api/SubscriptionResource.java:92]: Perhaps this should be a PUT instead?

Comment (foobar-1337) [reply src/main/java/se/sundsvall/disturbance/api/SubscriptionResource.java]: Fixed!

**Review by josc04 — APPROVED** [review]

**Pull request merged**